project-base/easy-coding-standard.yml

```diff
@@ -34,3 +34,7 @@ parameters:
 
         Shopsys\CodingStandards\Sniffs\ForbiddenDumpSniff:
             - '*/tests/ShopBundle/Functional/Model/Cart/CartFacadeTest.php'
+
+        Shopsys\CodingStandards\Sniffs\ForbiddenDoctrineInheritanceSniff:
```

Comment:
Shouldn't there be upgrade note for this change?

Reply:
you are right, I'll add them